pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -608,12 +608,11 @@ public CompletableFuture<Void> closeAsync() {
             }
 
             closeLocalMetadataStore();
+            if (configMetadataSynchronizer != null) {
+                configMetadataSynchronizer.closeAsync();
```

Comment:
The futures of the closeAsync operations should be added to the ` asyncCloseFutures` list, this is meaningful in some tests where flakiness is reduced by ensuring that everything has stopped.

Reply:
Done